<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinksDialog</class>
 <widget class="QWidget" name="LinksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>583</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="3">
    <widget class="QSplitter" name="msgSplitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="13">
        <widget class="QTreeWidget" name="linkTreeWidget">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <column>
          <property name="text">
           <string>Title / Comment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Score</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Peer / Link</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="expandButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../retroshare-gui/src/gui/images.qrc">
           <normaloff>:/images/edit_remove24.png</normaloff>:/images/edit_remove24.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sort by</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="rankComboBox">
         <item>
          <property name="text">
           <string>Combo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Time</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/kalarm.png</normaloff>:/images/kalarm.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ranking</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/records.png</normaloff>:/images/records.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>In last</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QComboBox" name="periodComboBox">
         <item>
          <property name="text">
           <string>Month</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/view_calendar_month.png</normaloff>:/images/view_calendar_month.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Week</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/view_calendar_week.png</normaloff>:/images/view_calendar_week.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Day</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/view_calendar_day.png</normaloff>:/images/view_calendar_day.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="7">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="8">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="3" column="9">
        <widget class="QComboBox" name="fromComboBox">
         <item>
          <property name="text">
           <string>All Peers</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/user/friends24.png</normaloff>:/images/user/friends24.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Own Links</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/user/identity16.png</normaloff>:/images/user/identity16.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="10">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="11">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item row="3" column="12">
        <widget class="QComboBox" name="topComboBox">
         <item>
          <property name="text">
           <string>Top 100</string>
          </property>
          <property name="icon">
           <iconset resource="../../retroshare-gui/src/gui/images.qrc">
            <normaloff>:/images/records.png</normaloff>:/images/records.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>101-200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>201-300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>301-400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>401-500</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bottom 100</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1" colspan="12">
        <widget class="QTextBrowser" name="linklabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextBrowser{border: 2px solid #CCCCCC; 
border-radius: 10px;
background: white;}</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Link:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_1">
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>311</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="anonBox">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Add Anonymous Link</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add Link/Comment</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="titleLineEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit{border: 2px solid #CCCCCC; 
border-radius: 10px;
background: white;}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Score:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="scoreBox">
           <item>
            <property name="text">
             <string>+2 Great!</string>
            </property>
            <property name="icon">
             <iconset resource="../../retroshare-gui/src/gui/images.qrc">
              <normaloff>:/images/filerating5.png</normaloff>:/images/filerating5.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>+1 Good</string>
            </property>
            <property name="icon">
             <iconset resource="../../retroshare-gui/src/gui/images.qrc">
              <normaloff>:/images/filerating4.png</normaloff>:/images/filerating4.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0 Okay</string>
            </property>
            <property name="icon">
             <iconset resource="../../retroshare-gui/src/gui/images.qrc">
              <normaloff>:/images/filerating3.png</normaloff>:/images/filerating3.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>-1 Sux</string>
            </property>
            <property name="icon">
             <iconset resource="../../retroshare-gui/src/gui/images.qrc">
              <normaloff>:/images/filerating2.png</normaloff>:/images/filerating2.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>-2 Bad Link</string>
            </property>
            <property name="icon">
             <iconset resource="../../retroshare-gui/src/gui/images.qrc">
              <normaloff>:/images/filerating1.png</normaloff>:/images/filerating1.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Url:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linkLineEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit{border: 2px solid #CCCCCC; 
border-radius: 10px;
background: white;}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="linkTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextEdit{border: 2px solid #CCCCCC; 
border-radius: 10px;
background: white;}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="titleBarFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="titleBarPixmap">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="linksCloud_images.qrc">:/images/irkick.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleBarLabel">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Links Cloud</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>596</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="addToolButton">
        <property name="text">
         <string>Add new link</string>
        </property>
        <property name="icon">
         <iconset resource="../../retroshare-gui/src/gui/images.qrc">
          <normaloff>:/images/edit_add24.png</normaloff>:/images/edit_add24.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>16</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="linksCloud_images.qrc"/>
  <include location="../../retroshare-gui/src/gui/images.qrc"/>
 </resources>
 <connections/>
</ui>
